test/box/ddl_lock_gh-2783.test.lua

```diff
@@ -15,19 +15,22 @@ _ = test_latch:create_index('primary', {unique = true, parts = {1, 'unsigned'}})
 fiber = require('fiber')
 c = fiber.channel(1)
 test_run:cmd("setopt delimiter ';'")
+-- Delay WAL so that creation of index won't be committed early
+box.error.injection.set('ERRINJ_WAL_DELAY', true);
 _ = fiber.create(function()
     test_latch:create_index("sec", {unique = true, parts = {2, 'unsigned'}})
     c:put(true)
 end);
 
--- Should be Ok for now
+-- Disable WAL delay right before creating another index so that the first
+-- DDL won't be committed when this one will start.
+-- Must fail with error - cannot create index when there is another space
+-- modification being committed.
 box.begin()
+    box.error.injection.set('ERRINJ_WAL_DELAY', false)
```

Comment:
Looks like this test is long obsolete. We already test parallel DDL execution in a bunch of other tests (e.g. `ddl_parallel`). Let's just drop it and add `gh-2783` comment to the `ddl_parallel` test case.

Reply:
Dropped the test, mentioned the issue from `ddl_parallel`.